### DISCUSSION TITLE
Header format

### DISCUSSION THREAD

**Discussion by Holzchopf** [flp-community-scripts/community-scripts#10] [Ideas]
Hi everyone!

I added a second variant to the header format working draft. It is based on Google's style guide, I didn't re-invent the wheel there ;-) I was looking into different standardized python style guides when I came across this one. I chose to add this one because it's simple and it looks like it will handle multi-line definitions better (for example the license text or a list of changes).

What do you think?

**Comment by fruitybagel** [comment]
I just came across this, hmm have a look at current archive formats what do you think? I also added a v3 variant, it's probably not super standard but I don't think it's heavy or anything or idk

**Reply by Holzchopf** [reply]
Looks good.

One little concern: why `"""flp` and not `"""flpianoroll`? FLP is also the abbreviation for the FL Studio project file format. And if IL gives us more python scripting abilities in FLS - a thing they said they're considering already - `flp` might become ambiguous.

**Reply by fruitybagel** [reply]
Can do `flpianoroll`, I was just thinking of something short quick. Maybe `"""flpp` ?

**Reply by Holzchopf** [reply]
Now that there are already scripts using `"""flp` I'd say we stick with that until we have a good reason to change it ;-)

**Reply by fruitybagel** [reply]
It would be fairly trivial to refactor it's just a matter to committing this to spec
